app/utils/formats.py

```diff
@@ -206,6 +206,74 @@ def from_donation(cls, item: ANHash) -> Optional[ATRecord]:
             custom={},
         )
 
+    @classmethod 
+    def from_mobilize(cls, data: Dict[str, str]) -> ATRecord:
+        timeslot_id = data["timeslot id"]
+        if timeslot_id == "":
+            data["shift id"] = data["email"] + "-" + data["signup created time"]
+        else: 
+            data["shift id"] = data["email"] + "-" + data["timeslot id"]
+
+        if data["attended"] == "":
+            data["attended"] = None
+        if data["rating"] == "":
+            data["rating"] = None
+        if data["Spanish"] == "":
+            data["Spanish"] = None
+        if data["status"] == "":
+            data["status"] = None
```

Comment:
It's not clear what you are trying to do here.  Are you trying to remove these keys from the data dictionary?  If so, you do that with `del data[key]`.  Also, whenever you're doing the same thing to a bunch of keys, it's clearer (and makes changing the keys easier) to use a loop.  For example, you could check and delete all four with this code:
```suggestion
for key in ("attended", "rating", "Spanish", "status"]:
    if not data.get(key):
        del data[key]       
```